core/src/test/scala/pureconfig/equality/package.scala

```diff
@@ -0,0 +1,26 @@
+package pureconfig
+
+import java.util.regex.Pattern
+
+import scala.util.matching.Regex
+
+import org.scalactic.Equality
+import org.scalactic.TypeCheckedTripleEquals._
+
+package object equality {
+
+  implicit val patternEquality = new Equality[Pattern] {
```

Comment:
Total nitpick: I'd normally mark these `final` and capitalize the first letter. I don't know that we have a consistent pattern on that in Pureconfig. 
Feel free to ignore me.